Anlab.Mvc/appsettings.json

```diff
@@ -22,6 +22,8 @@
     "StorageContainerName": "anlabupload", //Dev
     "CyberSourceUrl": "https://testsecureacceptance.cybersource.com/pay",
     "FinancialLookupUrl": "https://kfs.ucdavis.edu/kfs-prd/remoting/rest/fau",
+    "FinancialLookupUrlNewProd": "https://financials.api.adminit.ucdavis.edu/fau",
+    "FinancialLookupUrlNewUat": "https://financials-uat.api-np.adminit.ucdavis.edu/fau",
```

Comment:
Do we want to keep putting urls into source control, or should we start marking these as [external] and place the actual urls in 1password?

Reply:
Generally, but not always, we use External for secrets. Nothing secret about this.
I'm not going to merge this, just use it as a reminder for when we add an app setting,

Reply:
> I'm not going to merge this, just use it as a reminder for when we add an app setting,

Oh, now I get it.